utils/utils.js

```diff
@@ -16,4 +12,13 @@ exports.checkUserExists = (username) => {
     .then(({rows}) => {
         return rows.length === 1
     })
-}
\ No newline at end of file
+}
+
+exports.checkCommentExists = (comment_id) => {
+    return db.query('SELECT * FROM comments WHERE comment_id = $1',[comment_id])
+    .then(({rows}) => {
+        return rows.length === 1
+    })
+}
+
+//could potentially turn these 3 functions into one by inserting more parameters would require an overhaul of other modules in models
```

Comment:
We should delete comments. There is a function that gets articles by their id already :)